docs/4-auth.md

```diff
@@ -40,6 +40,11 @@ adminPassword: "<admin password>"
 adminUsername: "admin"
 # Configure zero or more authentication backends
 auth:
+  sessionStore:
+    # 32 random bytes used to sign session cookies. It's generated randomly
+    # if not present. Need to be set when running in HA setup (more than one replica)
+    # It has to be 32 bytes long (/[0-9a-f]{64}/)
+    secret: 4f68646565736f6f5368346f685468656567364c696537636569746861696368
```

Comment:
What do you think about this instead?
```suggestion
    # 32 random bytes in hexadecimal encoding (64 chars) used to sign session cookies. It's generated randomly
    # if not present. Need to be set when running in HA setup (more than one replica)
    secret: "<session store secret>"
```
The description might be a bit simpler than a regular expression.

Using placeholders in examples that are evident as placeholder also should help to prevent people copying/keeping the example, which would be less than ideal.

Reply:
yeah, good idea.